Add missing Expand shortcodes (#1867)
truenas/documentation: content/SCALE/SCALEUIReference/Virtualization/VirtualizationScreens.md

| **Shutdown Timeout** | Enter the time in seconds the system waits for the VM to cleanly shut down. During system shutdown, the system initiates power-off for the VM after the shutdown timeout entered expires. |
| **Start on Boot** | Select to start this VM when the system boots. |
| **Enable Display** | Enable a Display (Virtual Network Computing) remote connection. Requires UEFI booting. |

> | **Display Type** | Select either [VNC](https://novnc.com/info.html) or [SPICE](https://spice-space.org/) from the dropdown list. VNC is the most widely used option with the best display but is slower than SPICE. SPICE has faster data transfer speed but a lower quality display, and is not as secure as VNC.|

Author: ```suggestion
| **Display Type** | Select either [VNC](https://novnc.com/info.html) or [SPICE](https://spice-space.org/) from the dropdown list. VNC is the most widely used option with the best display, but it is slower than SPICE. SPICE has faster data transfer speed but a lower quality display and is not as secure as VNC.|
```

| **Cores** | Required. Enter the number of cores per virtual CPU socket. The product of vCPUs, cores, and threads must not exceed 16. |
| **Threads** | Required. Enter the number of threads per core. A single CPU core can have up to two threads per core. A dual core could have up to four threads. The product of vCPUs, cores, and threads must not exceed 16. |
| **Optional: CPU Set (Examples: 0-3,8-11)** | Specify the logical cores that VM is allowed to use. Better cache locality can be achieved by setting CPU set base on CPU topology. E.g. to assign cores: 0,1,2,5,9,10,11 you can write: `1-2,5,9-11` |

> | **Pin vcpus** | When number of vcpus is equal to number of cpus in CPU set vcpus can be automatically pinned into CPU set. Pinning is done by mapping each vcpu into single cpu number in following the order in CPU set. This improves CPU cache locality and can reduce possible stutter in GPU passthrough VMs. |

Author: ```suggestion
| **Pin vcpus** | When the number of vCPUs equals the number of CPUs in **CPU Set**, vCPUs can be automatically pinned into **CPU Set**. Pinning is done by mapping each vCPU into a single CPU number following the order in **CPU Set**. This improves CPU cache locality and can reduce possible stutter in GPU passthrough VMs. |
```

Author: Goodness, the UI tooltips are messy.

Reviewer: Yes they are! We need to start rewriting them so they make more sense!

| **CPU Model** | Select a CPU model to emulate. |
| **Memory Size** | Allocate RAM for the VM. Minimum value is 256 MiB. This field accepts human-readable input (Ex. 50 GiB, 500M, 2 TB). If units are not specified, the value defaults to bytes. |

> | **Optional: NUMA nodeset (Example: 0-1)** | Node set allows setting NUMA nodes for multi NUMA processors when CPU set was defined. Better memory locality can be achieved by setting node set based on assigned CPU set. Example: if cpus 0,1 belong to NUMA node 0, then setting nodeset to 0 improves memory locality. |

Author: ```suggestion
| **Optional: NUMA nodeset (Example: 0-1)** | Node set allows setting NUMA nodes for multi NUMA processors when CPU set was defined. Better memory locality can be achieved by setting nodeset based on the assigned CPU set. Example: if CPUs 0,1 belong to NUMA node 0, setting nodeset to 0 improves memory locality. |
```

| **Cores** | Required. Enter the number of cores per virtual CPU socket. The product of vCPUs, cores, and threads must not exceed 16. |
| **Threads** | Required. Enter the number of threads per core. A single CPU core can have up to two threads per core. A dual core could have up to four threads. The product of vCPUs, cores, and threads must not exceed 16. |
| **Optional: CPU Set (Examples: 0-3,8-11)** | Specify the logical cores that VM is allowed to use. Better cache locality can be achieved by setting CPU set base on CPU topology. E.g. to assign cores: 0,1,2,5,9,10,11 you can write: `1-2,5,9-11` |

> | **Pin vcpus** | When number of vcpus is equal to number of cpus in CPU set vcpus can be automatically pinned into CPU set. Pinning is done by mapping each vcpu into single cpu number in following the order in CPU set. This improves CPU cache locality and can reduce possible stutter in GPU passthrough VMs. |

Author: ```suggestion
| **Pin vcpus** | When the number of vCPUs equals the number of CPUs in **CPU Set**, vCPUs can be automatically pinned into **CPU Set**. Pinning is done by mapping each vCPU into a single CPU number following the order in **CPU Set**. This improves CPU cache locality and can reduce possible stutter in GPU passthrough VMs. |
```

| **CPU Model** | Select a CPU model to emulate. |
| **Memory Size** | Allocate RAM for the VM. Minimum value is 256 MiB. This field accepts human-readable input (Ex. 50 GiB, 500M, 2 TB). If units are not specified, the value defaults to bytes. |

> | **Optional: NUMA nodeset (Example: 0-1)** | Node set allows setting NUMA nodes for multi NUMA processors when CPU set was defined. Better memory locality can be achieved by setting node set based on assigned CPU set. Example: if cpus 0,1 belong to NUMA node 0, then setting nodeset to 0 improves memory locality. |

Author: ```suggestion
| **Optional: NUMA nodeset (Example: 0-1)** | Node set allows setting NUMA nodes for multi NUMA processors when CPU set was defined. Better memory locality can be achieved by setting nodeset based on the assigned CPU set. Example: if CPUs 0,1 belong to NUMA node 0, setting nodeset to 0 improves memory locality. |
```
